index.html

```diff
@@ -93,258 +96,275 @@ <h1 class="title">
               alt="image"
               class="grid-images"
             />
-            <h2 class="general-titles">Tonic</h2>
-            <ul class="span-grid">
-              <li class="color">CANOPY</li>
-              <img
-                src="./resources/work-section/icons/Counterdot.png"
-                alt=""
-                class="dot"
-              />
-              <li>Back End Dev</li>
-              <img
-                src="./resources/work-section/icons/Counterdot.png"
-                alt=""
-                class="dot"
-              />
-              <li>2015</li>
-            </ul>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="span-grid2">
-              <li>HTML</li>
-              <li>CSS</li>
-              <li>JavaScript</li>
-            </ul>
-            <button class="button-project">See Project</button>
+            <div class="Desktop-version">
+              <h2 class="general-titles">Tonic</h2>
+              <ul class="span-grid">
+                <li class="color">CANOPY</li>
+                <img
+                  src="./resources/work-section/icons/Counterdot.png"
+                  alt=""
+                  class="dot"
+                />
+                <li>Back End Dev</li>
+                <img
+                  src="./resources/work-section/icons/Counterdot.png"
+                  alt=""
+                  class="dot"
+                />
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="span-grid2">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+              <button class="button-project">See Project</button>
```

Comment:
- [ ]  I would kindly suggest you to add button interaction to all necessary button expecially see project button ,when the project is hovered on with the mouse the description and dark opacity disappears and a See Project button appears to the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository. Kindly fix this to make your website look user friendly and fit to the design according to the project requirement.